src/modules/ttlmonitor/test/TTLMonitorInstrumentTest.cpp

```diff
@@ -161,10 +159,8 @@ TEST_F(TTLMonitorInstrumentTest, BeamMonitorTOF) {
   ttlmonitor->ESSReadoutParser.Packet.Time.setReference(1, 100000);
   ttlmonitor->ESSReadoutParser.Packet.Time.setPrevReference(1, 0);
 
-  auto Readouts =
-      ttlmonitor->TTLMonParser.parse(ttlmonitor->ESSReadoutParser.Packet);
+  ttlmonitor->TTLMonParser.parse(ttlmonitor->ESSReadoutParser.Packet);
       counters.TTLMonStats = ttlmonitor->TTLMonParser.Stats;
-  ASSERT_EQ(Readouts, 1);
```

Comment:
Did asserting the result == 0 mean checking no errors had occurred? Can we replace this check in the tests with error counter checks?